experimenter/experimenter/nimbus-ui/src/components/PageEditAudience/FormAudience/index.test.tsx

```diff
@@ -511,11 +510,6 @@ describe("FormAudience", () => {
     expect(targetingConfigSlug.value).toEqual(
       MOCK_CONFIG!.targetingConfigs![1]!.value,
     );
-    expect(screen.getByTestId("isFirstRun")).toHaveProperty("checked", true);
-    expect(screen.getByTestId("isFirstRun")).toBeDisabled();
-    await expect(
-      screen.getByTestId("is-first-run-required-warning"),
-    ).toBeInTheDocument();
   });
 
```

Comment:
since you are removing this from here, I would say keep the nature of the test and modify it appropriately, that test is important, if the application is set up as desktop in the test, then change it to mobile and choose targeting/modify it accordingly

Reply:
@yashikakhurana okay mam i will modify the changes